[github-actions] test_dua_dad.py fails intermittently

**Describe the bug**

tests/scripts/thread-cert/backbone/test_dua_dad.py fails intermittently

**Console/log output**

- [logs_61772.zip](https://github.com/openthread/openthread/files/6165151/logs_61772.zip)
- [thread-1-2-backbone-results.zip](https://github.com/openthread/openthread/files/6165152/thread-1-2-backbone-results.zip)

Got error:
```
2021-06-24T17:41:51.6591916Z ----------------------------------------------------------------------
2021-06-24T17:41:51.6592392Z Traceback (most recent call last):
2021-06-24T17:41:51.6593225Z   File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/thread_cert.py", line 282, in tearDown
2021-06-24T17:41:51.6593889Z     self._verify_packets(test_info_path)
2021-06-24T17:41:51.6594748Z   File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/thread_cert.py", line 310, in _verify_packets
2021-06-24T17:41:51.6595379Z     self.verify(pv)
2021-06-24T17:41:51.6596033Z   File "./tests/scripts/thread-cert/backbone/test_dua_dad.py", line 222, in verify
2021-06-24T17:41:51.6597909Z     pv.verify_dua_registration(ROUTER1, ROUTER1_DUA, pbbr_eth=PBBR_ETH, pbbr_src64=PBBR, sbbr_eth=SBBR_ETH)
2021-06-24T17:41:51.6599124Z   File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/pktverify/packet_verifier.py", line 332, in verify_dua_registration
2021-06-24T17:41:51.6600338Z     assert bq1.sniff_timestamp - dr.sniff_timestamp <= 1.0, bq1.sniff_timestamp - dr.sniff_timestamp
2021-06-24T17:41:51.6600936Z AssertionError: 1.0004706382751465
2021-06-24T17:41:51.6601187Z 
```

--- a/tests/scripts/thread-cert/pktverify/packet_verifier.py
+++ b/tests/scripts/thread-cert/pktverify/packet_verifier.py
@@ -329,7 +329,7 @@ class PacketVerifier(object):
         bq1 = pkts.filter_backbone_query(dua, eth_src=pbbr_eth, port=BB).must_next()
         bq1_index = pkts.index
 
-        assert bq1.sniff_timestamp - dr.sniff_timestamp <= 1.0, bq1.sniff_timestamp - dr.sniff_timestamp
+        assert bq1.sniff_timestamp - dr.sniff_timestamp <= 1.01, bq1.sniff_timestamp - dr.sniff_timestamp
 
         # PBBR should send /b/bq for Router1's DUA (2nd time)
         bq2 = pkts.filter_backbone_query(dua, eth_src=pbbr_eth, port=BB).must_next()

